tox.ini

```
@@ -16,6 +16,10 @@ deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/requirements-extra.txt
 
+[testenv:WINDOWS]
+deps =
+    -r{toxinidir}/requirements-windows.txt
```

Looks like only the windows requirements get installed in the tox environment, we probably want all three sets.  So want to add lines 16&17 under deps here as well.  As it does not seem to append.
```suggestion
    -r{toxinidir}/requirements.txt
    -r{toxinidir}/requirements-extra.txt
    -r{toxinidir}/requirements-windows.txt
```

---

Done (via inheritance)